syntax = "proto3";
package message;//package message定义包名，类似于命名空间，避免消息命名冲突。
service VerifyService {   //service VerifyService：定义一个名为 VerifyService 的 gRPC 服务。rpc GetVerifyCode：定义了一个远程调用方法 GetVerifyCode。接收参数类型：GetVerifyReq（请求消息）。返回参数类型：GetVerifyRsp（响应消息）。
  rpc GetVerifyCode (GetVerifyReq) returns (GetVerifyRsp) {}
}
message GetVerifyReq { //GetVerifyReq：定义请求消息。string email = 1;：表示请求体包含一个字段 email，字段编号为 1。编号必须唯一，字段类型是字符串。
  string email = 1;
}
message GetVerifyRsp {
  int32 error = 1;
  string email = 2;
  string code = 3;
}